datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyPartitionAssignment.java

```diff
@@ -41,14 +47,32 @@
  * Tests for {@link StickyPartitionAssignmentStrategy}
  */
 public class TestStickyPartitionAssignment {
+  private EmbeddedZookeeper _embeddedZookeeper;
```

Comment:
Currently there are no tests for StickyPartitionAssignmentStrategyFactory. One important test can be to not set CFG_ZK_ADDRESS with elasticTaskCount enabled.

Reply:
I am checking this for the strategy itself. Now that the factory has been modified to always disable elastic task assignment if the zkAddress is empty, we should be good here.